datasets/br/br_tcu_debarred/crawler.py

```diff
@@ -0,0 +1,84 @@
+from zavod import Context, helpers as h
+
+def crawl_item(input_dict: dict, context: Context):
+    """
+    Creates an entity from the raw data.
+
+    :param input_dict: Data dict extracted from source.
+                       Should have at least following keys:
+                        - cpf_cnpj
+                        - nome
+                        - processo
+                        - deliberacao
+                        - data_transito_julgado
+                        - data_final
+    :param context: The context object.
+    """
+
+    entity = context.make("LegalEntity")
+
+    raw_tax_number = input_dict["cpf_cnpj"]
+
+    # If it's length 18 with punctuation, then it is a CNPJ (XX.XXX.XXX/XXXX-XX)
+    if len(raw_tax_number) == 18:
+        entity.add_schema("Company")
+        tax_number = h.clean_br_cnpj(raw_tax_number)
+        prefix = "br-cnpj"
+
+    # If it's length 14 with punctuation, then it is a CPF (XXX.XXX.XXX-XX)
+    elif len(raw_tax_number) == 14:
+        entity.add_schema("Person")
+        tax_number = h.clean_br_cpf(raw_tax_number)
+        prefix = "br-cnpj"
+
+    # If it's neither, then we just use the raw number
+    else:
+        context.log.warning("Entity type not defined by tax number")
+        tax_number = raw_tax_number
+        prefix = "br-unknown"
+
+    entity.id = context.make_slug(tax_number, prefix=prefix)
+    entity.add("name", input_dict["nome"])
+    entity.add("taxNumber", tax_number)
+    entity.add("country", "br")
+    entity.add("topics", "debarment")
```

Comment:
good. Please also add to the debarment collection

Reply:
Is `collection` another parameter for the entity different from the topics?

Reply:
oh, sorry, no a collection is a type of dataset that includes other datasets as children. 

This is a new section in the tutorial: https://zavod.opensanctions.org/tutorial/#add-your-crawler-to-a-collection